lib/steep/server/interaction_worker.rb

```diff
@@ -80,9 +80,16 @@ def process_hover(job)
             content = hover.content_for(path: job.path, line: job.line, column: job.column+1)
             if content
               range = content.location.yield_self do |location|
-                start_position = { line: location.line - 1, character: location.column }
-                end_position = { line: location.last_line - 1, character: location.last_column }
-                { start: start_position, end: end_position }
+                case content.location
```

Comment:
You can use `#as_lsp_range` (https://github.com/soutaro/steep/blob/master/lib/steep/range_extension.rb)

Reply:
thank you for the review!
updated it here https://github.com/soutaro/steep/pull/385/commits/d93aee587ef7710723f6f5791b8af0989d8682b7